client/src/Components/Pages/Search/Search.style.js

```diff
@@ -0,0 +1,168 @@
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+import { colors, shadows, organizations } from "./../../../theme";
+
+export const SearchWrapper = styled.div`
+  width: 100%;
+  max-width: 40rem;
+  margin: 1rem auto;
+  padding: 0 1rem;
+  padding-top: 2rem;
+  li {
+    list-style-type: none;
+  }
+`;
+
+export const HeadlineDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  padding-top: 2rem;
+  padding-bottom: 0.7rem;
+  color: ${colors.profileFontColor};
+  h2 {
+    font-size: 2rem;
+    font-weight: 350;
+  }
+  p {
+    font-style: italic;
+  }
+`;
+
+export const SearchLegendDiv = styled.div`
+  display: flex;
+  padding-left: 2rem;
+  flex-direction: column;
+  align-items: center;
+  padding-bottom: 1rem;
+
+  @media (min-width: 766px) {
+    padding-left: 6rem;
+  }
+`;
+
+export const RowDiv = styled.div`
+  display: flex;
+  width: 100%;
+`;
+
+export const ItemDiv = styled.div`
+  display: flex;
+  width: 50%;
+`;
+export const LegendTitle = styled.h2`
+  color: ${props => props.color};
+  font-size: 1rem;
+  margin-left: 0.25rem;
+`;
+
+export const SuggestionBox = styled.div`
+  border: 0.2px solid ${colors.lightGray};
+  box-shadow: ${colors.searchBoxShadow};
+  color: ${props => organizations[props.orgType].primary};
+`;
+
+export const AddItemBox = styled.div`
+  border: 0.2px solid ${colors.lightGray};
+  box-shadow: ${colors.searchBoxShadow};
+  color: black;
+  margin-top: -1rem;
+`;
+
+export const ReviewsFrame = styled.div`
+  display: flex;
+  justify-content: center;
+  padding-top: 0.3rem;
+  padding-left: 2rem;
+  color: ${props => organizations[props.orgType].primary};
+  @media (min-width: 766px) {
+    padding-left: 6rem;
+  }
+`;
+
+export const InnerDivLastReviews = styled.div`
+  text-align: left;
+  width: 100%;
+  display: flex;
+  @media (min-width: 766px) {
```

Comment:
theme breakpoint

Reply:
ha i will stop commenting now, just everywhere there is a `@media` just replace the number to use a variable from the theme file